samples/NativeLibrary/LoadLibrary.c

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//On unix make sure to compile using -ldl flag.
+
+//Set this value accordingly to your workspace settings
+#define PathToLibrary "./bin/Debug/netstandard2.0/linux-x64/native/NativeLibrary.so"
+
+
+#ifdef _WIN32
+#include "windows.h"
+#define symLoad GetProcAddress GetProcAddress
+#define libClose FreeLibrary
```

Comment:
The same concern about unloading applies to Windows as well. I would delete `libClose` completely, and just added this comment to where `libClose` is used today:

```
     // CoreRT libraries do not support unloading
     // See https://github.com/dotnet/corert/issues/7887
```